composer.json

```
@@ -18,7 +18,7 @@
         "drupal/checklistapi": "^2.0",
         "drupal/config_ignore": "2.3.0",
         "drupal/config_rewrite": "^1.4",
-        "drupal/core": "~9.2.20 || ~9.3.14",
+        "drupal/core": "~9.2.20 || ~9.3.16",
```

@panshulK This also needs to be changed:
```suggestion
        "drupal/core": "~9.2.21 || ~9.3.16",
```